pumpkin-protocol/src/client/play/c_command_suggestions.rs

```diff
@@ -0,0 +1,61 @@
+use std::{borrow::Cow, hash::Hash};
+
+use pumpkin_core::text::TextComponent;
+use pumpkin_macros::client_packet;
+
+use crate::{ClientPacket, VarInt};
+
+#[client_packet("play:command_suggestions")]
+pub struct CCommandSuggestions<'a> {
+    id: VarInt,
+    start: VarInt,
+    length: VarInt,
+    matches: Vec<CommandSuggestion<'a>>,
+}
+
+impl<'a> CCommandSuggestions<'a> {
+    pub fn new(
+        id: impl Into<VarInt>,
+        start: impl Into<VarInt>,
+        length: impl Into<VarInt>,
```

Comment:
This is against the design of Pumpkin, We use just VarInt's as args

Reply:
Thats quite unergonomic but I changed it